### DISCUSSION TITLE
Does the enforce_ssl configuration still exist?

### DISCUSSION THREAD

**Discussion by white-than-wood** [hexojs/hexo#4977] [Question / Answer] [answered]
Can I still configure enforce_ssl now????

**Comment by white-than-wood** [comment]
Any answers?

**Comment by SukkaW** [comment, marked as answer]
`enforce_ssl` no longer exists in Hexo, and you don't need this option.

**Reply by white-than-wood** [reply]
I see.

**Reply by SukkaW** [reply]
An explanation about why this option is no longer needed:

- If a URL is inlined by your theme's template, there is nothing Hexo can do: Hexo will always render the template as-is.
- If the theme template is using Hexo built-in `url_for` and `full_url_for` helper, Hexo will use the `url` and `root` from `_config.yml`.
- If you use `{% post_link %}` and `{% post_path %}` tag plugin in your posts, Hexo will also use the `url` and `root` from `_config.yml`.
- If you include an HTTP external link in your posts, Hexo shouldn't enforce HTTPS for it, as it is out of your control.
- So you only need to set an HTTPS url in your `_config.yml`, the `enforce_ssl` option is no longer required.
